assets/js/DataGrid.js

```diff
@@ -513,6 +534,10 @@ export default class DataGrid extends React.Component {
       console.log('More than one row changed at a time in DataGrid, how?')
     }
 
+    if(this.props.isReadOnly) {
+      return;
```

Comment:
We should never get here, right?

If that's the case, change `return` to `throw new Error(...)`